packages/playground/src/plugins/vuetify.ts

```diff
@@ -20,6 +20,7 @@ const vuetify = createVuetify({
         colors: {
           primary: "#7de3c8",
           secondary: "#14A1B8",
+          calcprimary: "#1AA18F",
```

Comment:
Please rename color to a more generic  name
if it won't be used anywhere else so delete it from vars and use it directly in component

Reply:
It will be used somewhere else so i'll leave it in the vars but change it to have a more generic name